### ISSUE TITLE
Resharper 2024.1.1 -> Incompatible with Unity Support

### ISSUE THREAD

**Issue by dcosmin-97** [JetBrains/resharper-unity#2442]
Hi!

The plugin is no longer compatible with the latest update of Resharper.
In the extension editor it shows as incompatible and the extension is no longer working.

Editor: Visual Studio 2022

Thank you!

**Comment by Villfuk02** [comment]
I have the same issue

**Comment by sr4dev** [comment]
You need to install the ReSharper C++.
https://github.com/JetBrains/resharper-unity/issues/2439#issuecomment-2066538825

**Comment by Villfuk02** [comment]
Thanks, that solved it!

**Comment by benblo** [comment]
This happened to me as well: Unity Support extension was stuck at 2023.1 and marked "incompatible" in the extension list, with no option to update, only option was to uninstall, which I did, and then it didn't show up in the extension list anymore (even after updating VS + reboot).

I had to download the nuget package (2024.3.0.191) and install it manually, which isn't exactly intuitive: add my download folder to the package sources etc.
It's working now, although I now have the "dictionary file" error when VS launches, which I used to have but hadn't seen in a long time ("Dictionary file 'C:\Users\Ben\AppData\Local\JetBrains\plugins\JetBrains.Unity.2023.3.2.100\DotFiles\Extensions\JetBrains.Unity\dictionaries\unity.dic' is not found on disk").

I'm on VS 2022 latest (17.12.3), Resharper latest (2024.3.2), Unity 6 (6000.0.25f1) with Visual Studio Editor package latest (2.0.22)
